src/components/map/components/DeparturesDialogSheet.tsx

```diff
@@ -162,6 +167,17 @@ const DeparturesDialogSheet = ({
         }}
       />
       <View style={styles.departuresContainer}>
+        {distance && (
+          <View style={styles.distanceLabel}>
+            <ThemeIcon
+              svg={walk}
+              fill={theme.text.colors.secondary}
+            ></ThemeIcon>
+            <ThemeText type="body__secondary" color="secondary">
+              {distance.toFixed() + ' m'}
```

Comment:
Looks like it's "200m" instead of "200 m" in the sketches.

```suggestion
              {distance.toFixed() + 'm'}
```

Reply:
https://www.sprakradet.no/sprakhjelp/Skriveregler/Mellomrom/
"Det skal vere mellomrom mellom tal og (forkorta) nemningar anten nemninga står framfor talet eller etter det.

10 kr, 5,68 kg, 25 l, 5 m, 10 min"

Reply:
Aha! In that case, we should update the front page as well:

![IMG_1897](https://user-images.githubusercontent.com/1774972/202441619-699491a8-fcee-41cd-a203-dfa861375ed4.PNG)

Reply:
https://github.com/AtB-AS/mittatb-app/pull/3063